css-positioning/absolute.css

```diff
@@ -0,0 +1,50 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+
+  /* your code here */
+
+  margin: 0;
+  position: relative;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+
+  /* your code here */
+
+  position: absolute;
+  bottom: 220px;
+  left: 235px;
```

Comment:
Woah, this is a _**big**_ movement. Instead of pushing it all the way from the `left` and `bottom`, you should try positioning it from the `right` and `top`